.github/workflows/lint-php-files.yml

```
@@ -0,0 +1,89 @@
+---
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    paths:
+      - 'src/**'
+      - 'tests/**'
+      - 'bin/trap'
+      - '.php-cs-fixer.dist.php'
+  push:
+    paths:
+      - 'src/**'
+      - 'tests/**'
+      - 'bin/trap'
+      - '.php-cs-fixer.dist.php'
+
+name: 🧹 Fix PHP coding standards
+
+jobs:
+  coding-standards:
+    timeout-minutes: 4
+    runs-on: ${{ matrix.os }}
+    concurrency:
+      cancel-in-progress: true
+      group: coding-standards-${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+        php-version:
+          - '8.2'
+        dependencies:
+          - locked
+    permissions:
+      contents: write
+    steps:
+      - name: ⚙️ Set git to use LF line endings
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
+
+      - name: 🛠️ Setup PHP
+        uses: shivammathur/setup-php@2.30.4
+        with:
+          php-version: ${{ matrix.php-version }}
+          extensions: none, ctype, dom, json, mbstring, phar, simplexml, tokenizer, xml, xmlwriter, sockets
+          ini-values: error_reporting=E_ALL
+          coverage: none
+
+      - name: 📦 Check out the codebase
+        uses: actions/checkout@v4.1.5
+
+      - name: 🛠️ Setup problem matchers
+        run: |
+          echo "::add-matcher::${{ runner.tool_cache }}/php.json"
+
+      - name: 🤖 Validate composer.json and composer.lock
+        run: composer validate --ansi --strict
+
+      - name: 🔍 Get composer cache directory
+        uses: wayofdev/gh-actions/actions/composer/get-cache-directory@v3.1.0
+
+      - name: ♻️ Restore cached dependencies installed with composer
+        uses: actions/cache@v4.0.2
+        with:
+          path: ${{ env.COMPOSER_CACHE_DIR }}
+          key: php-${{ matrix.php-version }}-composer-${{ matrix.dependencies }}-${{ hashFiles('composer.lock') }}
+          restore-keys: php-${{ matrix.php-version }}-composer-${{ matrix.dependencies }}-
+
+      - name: 📥 Install "${{ matrix.dependencies }}" dependencies with composer
+        uses: wayofdev/gh-actions/actions/composer/install@v3.1.0
+        with:
+          dependencies: ${{ matrix.dependencies }}
+
+      - name: 🛠️ Prepare environment
+        run: make prepare
+
+      - name: 🚨 Run coding standards task
+        run: composer cs:fix
+        env:
+          PHP_CS_FIXER_IGNORE_ENV: true
+
+      - name: 📤 Commit and push changed files back to GitHub
+        uses: stefanzweifel/git-auto-commit-action@v5.0.1
+        with:
+          commit_message: 'style(php-cs-fixer): lint php files and fix coding standards'
+          branch: ${{ github.head_ref }}
```

```suggestion
      - name: 📤 Commit and push changed files back to GitHub
        uses: stefanzweifel/git-auto-commit-action@v5.0.1
        with:
          commit_message: 'style(php-cs-fixer): lint php files and fix coding standards'
          branch: ${{ github.head_ref }}
          commit_author: "github-actions <github-actions@users.noreply.github.com>"
```

To avoid unverified (unsigned) commits

---

Fixed